chef/cookbooks/pacemaker/attributes/default.rb

```diff
@@ -15,15 +15,19 @@
 
 case node.platform
 when 'suse'
-  default[:pacemaker][:platform][:packages] = %w(pacemaker crmsh)
+  # Install xorg-x11-xauth by default so that 'crm configure graph'
+  # (which uses dotty) will show a nice configuration graph over ssh
+  # with X11 forwarding.
+  default[:pacemaker][:platform][:packages] = %w(
+    pacemaker crmsh xorg-x11-xauth
```

Comment:
Hmm.  It hasn't caused any extra in my testing, but that's probably because those packages are already part of the base SP3 image which mkcloud uses.  Our default autoyast profile uses the Minimal and base patterns, neither of which install X11 packages AFAICS.  I guess you are implying there's an argument that some customers might want to keep the install stripped to the bare minimum?